app/views/users/index.html.erb

```diff
@@ -1,12 +1,22 @@
-<main>
-
-<div class="user" id="1">
-    <img src="" alt"">
-    <div>
-       <h1>User name</h1>
-       <p>Number of post</p>
-        <%=@placeholder%>
-    </div>
-</div>
-
-</main>
\ No newline at end of file
+<% if @users.blank? %>
+  <p>There are not any users currently in the system.</p>
+<% else %>
+  <h1>All users</h1>
+  <main>
+    <ul>
+      <% @users.each do |user| %>
+        <a class="link" href="/users/<%= user.id %>">
+          <li class="list_item">
+            <div class="photo">
+              <%= image_tag (user.photo) %>
+            </div>
+            <div class="user_name">
+              <h3><%= user.name %></h3>
+              <p>Number of posts: <%= user.postsCounter %></p>
+            </div>
+          </li>
+        </a>
+      <% end %>
```

Comment:
- [ ] [OPTIONAL] I would strongly encourage you the usage of a partial, in instead of using this `@user.each` 
```ruby
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```